### ISSUE TITLE
[Proposal] Latest release

### ISSUE THREAD

**Issue by janezlapajne** [ladisk/pyFRF#10]
On Linkedin I saw that the new release was recently published. Personally, I don't use the package, but I had a quick glance at the repo. I noticed that the release itself is not labeled as "release", but just with ordinary tag (this could be easily changed through GitHub). Also, since I know that the Ladisk lab is research oriented, maybe you can add a citation file, so others can cite your work (see: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files). 

Anyways, great work!

**Comment by jankoslavic** [comment]
Thank you @janezlapajne! We will take care of this issue!

**Comment by jankoslavic** [comment]
@klemengit what is your opinion on the release issue here?

**Comment by janezlapajne** [comment]
Returning to this post for self-note. I came across this tool which can be of help: https://bit.ly/cffinit

**Comment by jankoslavic** [comment]
@janezlapajne thank you for the reminder. 
I have prepared a PR here: https://github.com/ladisk/pyFRF/pull/13
Have you had something like this in mind?

Have you any experience with CFF? How do you see https://zenodo.org vs cffinit?

**Comment by janezlapajne** [comment]
Yes, it looks good to me. No, I haven't tried the cffinit myself yet (still waiting for my latest submission to go through). However, I saw its utilization in one "serious" publication and it seemed cool to share. Regarding zenodo and cff - I think the best is to use a combination of both as it can be seen here: [zenodo](https://help.zenodo.org/faq/) under GitHub section. I think the main thing about cff is that the button for citation is generated so someone who finds the repo just clicks on it and the citation is easily copied. If you publish the GitHub repo on zenodo you can add the link of zenodo DOI directly to cff file.

**Comment by jankoslavic** [comment]
Thanks!

**Comment by jankoslavic** [comment]
Both suggestions are in place: the repo has a CITATION.cff, and the latest version (v1.2.0) is published as a proper GitHub Release (not just a tag). Thanks for the nudge — closing.
